app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -887,6 +887,14 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = true
     )
 
+    /**
+     * Indicates if the onboarding features should be shown.
+     */
+    var shouldShowOnboardingFeatures by booleanPreference(
```

Comment:
will this control all of _onboarding_, or just the new enhancements?

Reply:
Currently in scope are Sync CFR and JumpBackIn CFR

Reply:
Using just one shared preference and nimbus experiment for multiple features may cause issues in the scenario of the CFRs having different preconditions to be shown and based on https://mozilla-hub.atlassian.net/browse/FNXV2-20988 looks like this is the case for these two CFRs.
If only one flag controls multiple CFRs and not all will be shown at the exact same time based on just one flag check then when the first cfr is shown and persists that onboarding was shown (for it) no other onboarding CFRs will be shown afterwards if they check the same flag which now says that onboarding was shown.

Seems like using a Nimbus experiment to control each CFR is part of the requirements of each CFR ticket so maybe it would be best to leave it for each CFR implementation to add it's own feature flag?

Reply:
Using just one shared preference and nimbus experiment for multiple features may cause issues in the scenario of the CFRs having different preconditions to be shown and based on https://mozilla-hub.atlassian.net/browse/FNXV2-20988 looks like this is the case for these two CFRs.
If only one flag controls multiple CFRs and not all will be shown at the exact same time based on just one flag check then when the first cfr is shown and persists that onboarding was shown (for it) no other onboarding CFRs will be shown afterwards if they check the same flag which now says that onboarding was shown.

Seems like using a Nimbus experiment to control each CFR is part of the requirements of each CFR ticket so maybe it would be best to leave it for each CFR implementation to add it's own feature flag?

Reply:
I was under the impression they both will be behind the same feature flag and so I will implement another setting different for both CFRs. When sync CFR has been shown, the setting to it will be made false, however no change in the onboarding setting flag